package.json

```diff
@@ -6,10 +6,10 @@
   "scripts": {
     "release": "semantic-release",
     "lint": "eslint --max-warnings 0 --config .eslintrc lib",
-    "package": "npm run test && ncc build lib/index.js -o dist",
+    "package": "ncc build lib/index.js -o dist -s --external=\"@npmcli/run-script\"",
```

Comment:
Only for clarification, why `@npmcli/run-script` is treated as external? I think that we should also add `--minify` flag to minify output.

Reply:
> Only for clarification, why @npmcli/run-script is treated as external

for some reason it is not working in ncc, there is a bug that is not fixed and this was the only known workaround to add it as dependency and use this way

> I think that we should also add --minify flag to minify output.

I don't think it is necessary, it will only make it unreadable. It is not officially recommended by GH Actions, meaning it is not needed for them aka not needed for us

Reply:
> I don't think it is necessary, it will only make it unreadable.

Do you really think that someone will read the `dist` bundle, even if it isn't minified? 😄 But ok, we can disable this at the moment.